recipe/meta.yaml

```diff
@@ -23,6 +39,11 @@ requirements:
         - ocl-icd  # [linux]
         - khronos-opencl-icd-loader  # [win]
         - mingwpy  # [win]
+{% if osx_variant == "pocl" %}
+        - osx-pocl-opencl
+        - pocl
```

Comment:
Asking out of ignorance. Would it make sense to have `pocl` as an option on Linux?